collect_app/src/main/java/org/odk/collect/android/views/MediaLayout.java

```diff
@@ -302,6 +304,8 @@ public void onClick(View v) {
 
         // Determine the layout constraints...
         // Assumes LTR, TTB reading bias!
```

Comment:
This comment is no longer accurate! How about changing it to "Determine the layout constraints. Right-to-left support works for API > 16."